app/src/main/java/com/example/todo/HomePage.java

```diff
@@ -22,6 +23,7 @@
 import android.view.View;
 import android.widget.EditText;
 import android.widget.ImageView;
+import android.widget.Toast;
```

Comment:
Please review the activity if Toast is used or not. If it hasn't been used, remove this import. It seems to have made its way here from the debug and test cycle.